libs/ui/src/text.test.tsx

```diff
@@ -114,31 +113,6 @@ describe('renders Text', () => {
         text={'I’m a paragraph.\n\nAnd I’m a paragraph with a\nline break.'}
       />
     );
-    expect(container).toMatchInlineSnapshot(`
-      @media print {
-
-      }
-
-      <div>
-        <p
-          class=""
-        >
-          <span>
-            I’m a paragraph.
-          </span>
-        </p>
-        <p
-          class=""
-        >
-          <span>
-            And I’m a paragraph with a
-          </span>
-          <br />
-          <span>
-            line break.
-          </span>
-        </p>
-      </div>
-    `);
+    expect(container).toMatchSnapshot();
```

Comment:
This should be replaced with test assertions which look for the content to be broken into specific container tags. But doesn't have to be changed for this PR.

Reply:
I'll improve it in this PR.